app/routes.php

```diff
@@ -63,7 +63,7 @@ function (RouteCollectorProxy $paymentMethodsGroup) {
     });
 
     // Authenticated through Stripe's SDK signature verification
-    $app->post('/hooks/stripe', Hooks\StripeChargeUpdate::class);
+    $app->post('/hooks/stripe', Hooks\StripePaymentsUpdate::class);
```

Comment:
renamed the class as we're now going to be taking events related to both charges and payment intents.